frontends/web/src/components/devices/bitbox02/checkbackup.tsx

```diff
@@ -31,21 +32,24 @@ type Props = CheckProps & TranslateProps;
 interface State {
     activeDialog: boolean;
     message: string;
+    foundBackup?: Backup;
 }
 
 class Check extends Component<Props, State> {
     public state = {
         activeDialog: false,
         message: '',
+        foundBackup: undefined,
     };
 
-    private checkBackup = () => {
+    private checkBackup = backups => {
         apiPost('devices/bitbox02/' + this.props.deviceID + '/backups/check', {
             silent: false,
         }).then(({ backupID, success }) => {
             let message;
             if (success && backupID) {
-                message = this.props.t('backup.check.success', { name: backupID });
+                this.setState({foundBackup: backups.find(backup => backup.id === backupID)});
```

Comment:
if backup could not be found, you will be showing an empty dialog. probably better to show the notOK message if that happens, or even better `t('unknownError', { errorMessage: "not found" })`

Reply:
The notOK message is already shown when a matching backup is not found, see the else statement on lines 53-4 after your comment: 
```
} else {
    message = this.props.t('backup.check.notOK');
}
```

Reply:
No I meant if the device returns a found backup ID, but we can't find it in the backups list. `foundBackup` can be undefined in the line 51.

Reply:
Gotcha, now the `unknownError` makes more sense :)
I'll make a fix